src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -96,26 +96,29 @@ public function prepare_items() {
 		}
 
 		$args['posts_per_page'] = $per_page;
-
-		if ( 'scheduled' === $this->tab->get_slug() && ! empty( $_REQUEST['s'] ) ) {
+		$search_term = tribe_get_request_var( 's' );
+		if ( 'scheduled' === $this->tab->get_slug() && ! empty( $search_term ) ) {
 			// nonce check if search form submitted.
 			$nonce = isset( $_POST['s'] ) && isset( $_POST['aggregator']['nonce'] ) ? sanitize_text_field( $_POST['aggregator']['nonce'] ) :  '';
-			if ( wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
+			if ( isset( $_GET['s'] ) || wp_verify_nonce( $nonce, 'aggregator_' . $this->tab->get_slug() . '_request' ) ) {
+				$search_term = filter_var( $search_term, FILTER_VALIDATE_URL ) 
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space

<sub>hash: 3dfccf4c54830808f7a144ba481924ae</sub>